src/command/marian_conv.cpp

```diff
@@ -13,10 +13,9 @@ int main(int argc, char** argv) {
   cli.add<std::string>("--from,-f", "Input model", "model.npz");
   cli.add<std::string>("--to,-t", "Output model", "model.bin");
 
-  cli.parse(argc, argv);
-
-  auto modelFrom = cli.get<std::string>("from");
-  auto modelTo = cli.get<std::string>("to");
+  auto opts = cli.parse(argc, argv);

```

Comment:
Similar to a separate comment by Marcin, I hope we can use the same mecahnism everywhere.

Would it be so bad to do something like this:

    auto opts = Options(cli.parse(argc, argv));
    auto modelFrom = opts->get<string>("from");